File: Common/Messages/PushNotification/ListPushNotificationMessageResponseModel.cs
using System;
using IOBootstrap.NET.Common.Messages.Base;
using IOBootstrap.NET.Common.Models.PushNotification;

namespace IOBootstrap.NET.Common.Messages.PushNotification;

public class ListPushNotificationMessageResponseModel : IOResponseModel
{

    public int Count { get; set; }
    public IList<PushNotificationMessageModel> Messages { get; set; }

    public ListPushNotificationMessageResponseModel(int count, IList<PushNotificationMessageModel> messages) : base()
    {
        this.Count = count;
        this.Messages = messages;
    }
}

File: React.UI/modules/iobootstrap-bo-pushnotifications/src/models/ListPushNotificationMessageResponseModel.ts
import { BaseResponseModel } from "iobootstrap-ui-base";
import PushNotificationMessageModel from "./PushNotificationMessageModel";

class ListPushNotificationMessageResponseModel extends BaseResponseModel {

    count: number;
    messages: PushNotificationMessageModel[];

    constructor() {
        super();

        this.count = 0;
        this.messages = [];
    }
}

export default ListPushNotificationMessageResponseModel;
